app/admin/models/Location_areas_model.php

```diff
@@ -3,19 +3,40 @@
 namespace Admin\Models;
 
 use Igniter\Flame\Database\Traits\Sortable;
+use Igniter\Flame\Database\Traits\Validation;
 use Igniter\Flame\Location\Models\AbstractArea;
 
 /**
  * Location areas Model Class
  */
 class Location_areas_model extends AbstractArea
 {
+    use Validation;
     use Sortable;
 
     const SORT_ORDER = 'priority';
 
     protected $fillable = ['area_id', 'type', 'name', 'boundaries', 'conditions', 'is_default', 'priority'];
 
+    public $rules = [
```

Comment:
So is the plan going forward to use rules on the model instead of Form requests? or both or ... ?

Reply:
To use both... But we still have the LocationAreas request class.

Reply:
Ok - is there any way of avoiding the duplication of the rules in both places? I can see them getting out of sync.